src/table/src/requests.rs

```diff
@@ -93,8 +96,20 @@ impl TryFrom<&HashMap<String, String>> for TableOptions {
                 .into();
             options.ttl = Some(ttl_value);
         }
+        if let Some(compaction_time_window) = value.get(COMPACTION_TIME_WINDOW_KEY) {
+            options.compaction_time_window = compaction_time_window
+                .parse::<i64>()
```

Comment:
From the original issue this `compaction_time_window` should be parsed as a duration string?

Reply:
Hi @waynexia 
thanks for raising this!
As far as I understand having `compaction_time_window` in the format of a duration string is still the end goal but it would be implemented as a part of another ticket.
@v0y4g3r suggested keeping it simple for now.